plugins/woocommerce/client/blocks/assets/js/blocks/product-details/utils.ts

```diff
@@ -2,9 +2,42 @@
  * External dependencies
  */
 import { PartialProduct, ProductDimensions } from '@woocommerce/data';
+import { getSetting } from '@woocommerce/settings';
 import { isEmpty } from '@woocommerce/types';
 import { __ } from '@wordpress/i18n';
 
+/**
+ * Check if WordPress version is 6.9 or later
+ */
+const isWP69OrLater = (): boolean => {
+	const wpVersion = getSetting( 'wpVersion' );
+	if ( ! wpVersion ) {
+		return false;
+	}
+	const [ major, minor ] = wpVersion.split( '.' ).map( Number );
+	return major > 6 || ( major === 6 && minor >= 9 );
+};
+
+/**
+ * Get accordion block names based on WordPress version
+ */
+const getAccordionBlockNames = () => {
+	if ( isWP69OrLater() ) {
```

Comment:
We have a JS util for this too, like it's used [here](https://github.com/woocommerce/woocommerce/blob/3a57a10fb346e02aab1c6cecfb9a8064ec7d275a/packages/js/block-templates/src/utils/register-woo-block-type.ts#L89), so you shouldn't need the custom function. 🙂